### PR TITLE
Clean code brian

### PR DESCRIPTION
**Pull request by BrianLa0616**
### clean code

- Removed duplicate/unused imports
- Used Carnelian in nav_bar like in other dart files

### CODE REVIEW

File: game/lib/widget/nav_bar.dart [hunk 1]
```diff
@@ -2,13 +2,10 @@ import 'dart:io';
 
 import 'package:flutter/material.dart';
 import 'package:game/events_leaderboard/events_leaderboard_widget.dart';
-import 'package:game/feedback/feedback.dart';
```
[the remark this answers is not in the document]
Comment (BrianLa0616) [reply game/lib/widget/nav_bar.dart:5]: I saw that feedback isn't implemented anywhere, so I think you're right. I added it back in for future use.

[the remark this answers is not in the document]
Comment (BrianLa0616) [reply game/lib/widget/nav_bar.dart:5]: I think you're right. I don't see feedback used anywhere, so it'll be probably used in the future. I added it back in!

[the remark this answers is not in the document]
Comment (BrianLa0616) [reply game/lib/widget/nav_bar.dart:5]: I think you're right. Feedback isn't implemented anywhere, so it'll probably be added later in nav_bar. I've added it back in.

File: game/lib/widget/leaderboard_user_cell.dart [hunk 2]
```diff
@@ -7,8 +7,6 @@ Widget leaderBoardUserCell(
       TextStyle(fontWeight: FontWeight.bold, fontSize: 40, color: Colors.white);
   var totalPosStyle =
       TextStyle(fontWeight: FontWeight.bold, fontSize: 20, color: Colors.white);
-  var nameStyle =
```
[the remark this answers is not in the document]
Comment (BrianLa0616) [reply game/lib/widget/leaderboard_user_cell.dart:10]: Good point!

**Review by nirbhaysinghnarang — COMMENTED** [review]
Good work going through this! A lot of these changes were long due lol. I think we should definitely work on building a ListTile widget like Youssef said, in order to further reduce code repetition. Also, we could create global text styles for specific font types and import them as necessary to not only avoid code duplication but also have a more consistent design schema. What do you think?

**Pull request merged**